api/client/swarm/opts.go

```diff
@@ -30,9 +30,11 @@ const (
 
 var (
 	defaultPolicies = []swarm.Policy{
-		{Role: WORKER, Autoaccept: true},
-		{Role: MANAGER, Autoaccept: false},
+		{Role: worker, Autoaccept: true},
+		{Role: manager, Autoaccept: false},
 	}
+
+	exists = struct{}{}
```

Comment:
You can just use `struct{}{}` directly instead of defining a variable for it. It doesn't matter either way, but that's more typical usage.